test_unstructured_client/integration/test_decorators.py

```diff
@@ -12,20 +12,22 @@
 
 @pytest.mark.parametrize("call_threads", [1, 2, 5])
 @pytest.mark.parametrize(
-    "filename, expected_ok",
+    ("filename", "expected_ok", "strategy"),
     [
-        ("_sample_docs/list-item-example-1.pdf", True),  # 1 page
-        ("_sample_docs/layout-parser-paper-fast.pdf", True),  # 2 pages
-        ("_sample_docs/layout-parser-paper.pdf", True),  # 16 pages
-        ("_sample_docs/fake.doc", True),
+        ("_sample_docs/list-item-example-1.pdf", True, "fast"),  # 1 page
+        ("_sample_docs/layout-parser-paper-fast.pdf", True, "fast"),  # 2 pages
+        # NOTE(mike): using "fast" strategy fails on this file for unknown reasons
```

Comment:
see this post on Slack: https://unstructuredai.slack.com/archives/C04TKD85KC6/p1715777702633619

Changing "fast" to "hi_res" fixed the issue **(which deserves a separate PR by the way❗)**

Reply:
done, removed